packages/editor-ui/src/components/Node.vue

```diff
@@ -84,9 +91,21 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 
 			return workflowResultDataNode.length;
 		},
+		nodeOffsetPosition() {
+			return this.$store.getters.getNodeViewOffsetPosition;
+		},
 		isExecuting (): boolean {
 			return this.$store.getters.executingNode === this.data.name;
 		},
+		isSingleActiveTriggerNode (): boolean {
+			return this.$store.getters.getActiveWorkflowTriggerNodes.length === 1 && !this.data.disabled;
+		},
+		isTriggerNode (): boolean {
+			return this.nodeType !== null ? this.nodeType.group.includes('trigger') : false;
+		},
+		isTriggerNodeTooltipEmpty () : boolean {
+			return this.nodeType !== null ? this.nodeType.eventTriggerDescription === '' : false;
```

Comment:
nitpick - can simplify a bit
```suggestion
		isTriggerNodeTooltipEmpty () : boolean {
			return !(this.nodeType && this.nodeType.eventTriggerDescription);
```
or even remove negation completely
```suggestion
		showTriggerNodeTooltip () : boolean {
			return this.nodeType && this.nodeType.eventTriggerDescription;
```
as with nitpicks always, it's up to you whether to make this change

Reply:
I did not change this one :)